packages/mesos/buildinfo.json

```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "5ce334c9b83a682e5a07b726e4aee430d4bc2a65",
-    "ref_origin" : "dcos-mesos-master-d9c90bf1"
+    "ref": "4fd55cefc70fb7e42def437cec594bebb8ed90a7",
+    "ref_origin" : "dcos-mesos-master-fc48ed69"
```

Comment:
Please use the `1.4.x` branch now that it has been created, preferably at the latest SHA (6241f77), which would be a new `dcos-mesos-1.4.x-6241f77` branch.